apps/api/v2/src/modules/organizations/teams/verified-resources/org-teams-verified-resources.controller.ts

```diff
@@ -0,0 +1,242 @@
+import { VerifyEmailInput } from "@/modules/verified-resources/inputs/verify-email.input";
+import { VerifyPhoneInput } from "@/modules/verified-resources/inputs/verify-phone.input";
+import { RequestEmailVerificationOutput } from "@/modules/verified-resources/outputs/request-email-verification-output";
+import { RequestPhoneVerificationOutput } from "@/modules/verified-resources/outputs/request-phone-verification-output";
+import {
+  VerifiedEmailOutput,
+  VerifiedEmailOutputData,
+  VerifiedEmailsOutput,
+} from "@/modules/verified-resources/outputs/verified-email.output";
+import {
+  VerifiedPhoneOutput,
+  VerifiedPhoneOutputData,
+  VerifiedPhonesOutput,
+} from "@/modules/verified-resources/outputs/verified-phone.output";
+import { VerifiedResourcesService } from "@/modules/verified-resources/services/verified-resources.service";
+import {
+  Body,
+  Controller,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Param,
+  ParseIntPipe,
+  Post,
+  Query,
+  UseGuards,
+} from "@nestjs/common";
+import { ApiHeader, ApiOperation, ApiTags } from "@nestjs/swagger";
+import { plainToClass } from "class-transformer";
+
+import { ERROR_STATUS, SUCCESS_STATUS } from "@calcom/platform-constants";
+import { SkipTakePagination } from "@calcom/platform-types";
+
+@Controller({
+  path: "/v2/organizations/:orgId/teams/:teamId/verified-resources",
+})
+@UseGuards(ApiAuthGuard, IsOrgGuard, RolesGuard, IsTeamInOrg, PlatformPlanGuard, IsAdminAPIEnabledGuard)
+@ApiTags("Organization Team Verified Resources")
+export class OrgTeamsVerifiedResourcesController {
+  constructor(private readonly verifiedResourcesService: VerifiedResourcesService) {}
+  @ApiOperation({
+    summary: "Request Email Verification Code",
+    description: `Sends a verification code to the email.`,
+  })
+  @Roles("TEAM_ADMIN")
+  @Throttle({
+    limit: 3,
+    ttl: 60000,
+    blockDuration: 60000,
+    name: "org_teams_verified_resources_emails_requests",
+  })
+  @PlatformPlan("ESSENTIALS")
+  @ApiHeader(API_KEY_OR_ACCESS_TOKEN_HEADER)
+  @Post("/emails/request")
+  @HttpCode(HttpStatus.OK)
+  async requestEmailVerificationCode(
+    @Body() body: RequestEmailVerificationInput,
+    @GetUser("username") username: string,
+    @GetUser("locale") locale: string
+  ): Promise<RequestEmailVerificationOutput> {
+    const verificationCodeRequest = await this.verifiedResourcesService.requestEmailVerificationCode(
+      { username, locale },
+      body.email
+    );
+
+    return {
+      status: verificationCodeRequest ? SUCCESS_STATUS : ERROR_STATUS,
+    };
+  }
+
+  @ApiOperation({
+    summary: "Request Phone Number Verification Code",
+    description: `Sends a verification code to the phone number.`,
+  })
+  @ApiHeader(API_KEY_OR_ACCESS_TOKEN_HEADER)
+  @Roles("TEAM_ADMIN")
+  @Throttle({ limit: 3, ttl: 60000, blockDuration: 60, name: "org_teams_verified_resources_phones_requests" })
+  @Post("/phone-numbers/request")
```

Comment:
I think it is fine if we use `phone` instead of `phone-numbers` because there is nothing more than phone number to veryify about a phone.

Next, just like with emails - word "request" is bit vague in my opinion. How about we call this endpoint `/phones/verification-code`.

And then rename `@Post("/phone-numbers/verify")` below to `@Post("/phones/verification-code/verify")`